components/style/parser.rs

```diff
@@ -81,3 +81,8 @@ impl<'a> ParserContext<'a> {
 pub fn log_css_error(input: &mut Parser, position: SourcePosition, message: &str, parsercontext: &ParserContext) {
     parsercontext.error_reporter.report_error(input, position, message);
 }
+
+/// XXXManishearth use this for everything
+pub trait Parseable {
```

Comment:
I don't know if this is a general rule, but it seems that many Rust trait names use imperative verbs rather than adjectives describing their ability, so maybe this should be `Parse` rather than `Parseable`?